### PR TITLE
#163246804 Sets up the database

### PR DESCRIPTION
**Pull request by MainaEK**
**What does this PR do?**
This PR sets up the database

**Description of the Task to be completed**
• Setting up the PostgreSQL database

**How should this be manually tested?**
• clone this repository to your local machine:
 https://github.com/erick-maina/Questioner_API.git
• branch into develop
$ git checkout develop
• run postgresql
$ psql postgres

**Pivotal tracker story associated with this PR:**
#163246804

**screenshot**
screenshot from 2019-01-16 10-36-31

### CODE REVIEW

**Comment by wathigo** [comment]
Great work Eric

File: app/db_con.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+DB_HOST =
+DB_USERNAME = 
+DB_PASS = 
+DB_NAME =
+DB_PORT =
+
+url = 
+
+con = psycopg2.connect(url)
```
Comment (ronaldndirangu) [inline app/db_con.py]: Add an end of line here, also where do the values for connection come from?

**Review by ronaldndirangu — CHANGES_REQUESTED** [review]

File: app/db_con.py (hunk 1 above)
Comment (MainaEK) [reply app/db_con.py]: I only set up PostgreSQL and psycopg2. I'll fill in those fields when I make the connection between the two and start using my endpoints through psycopg2 to edit the database

Comment (ronaldndirangu) [reply app/db_con.py]: Okay noted.

**Pull request merged**